crates/sncast/src/starknet_commands/show_config.rs

```diff
@@ -28,7 +28,7 @@ pub async fn show_config(
         None
     };
 
-    let rpc_url = Some(show.rpc.url.clone().unwrap_or(cast_config.url)).filter(|p| !p.is_empty());
+    let rpc_url = show.rpc.url.clone().or(cast_config.url);
```

Comment:
This chain returns an `Option`, is a case where both rpc url and config_url are none is valid?